st2api/st2api/controllers/v1/triggers.py

```diff
@@ -395,7 +388,13 @@ def get_all(self, **kw):
         return trigger_instances
 
     def _get_trigger_instances(self, **kw):
-        kw['limit'] = int(kw.get('limit', self.default_limit))
+        kw['limit'] = int(kw.get('limit') or self.default_limit)
```

Comment:
Ah, so it looks like now it contains `None` if value is not supplied by the user, or?